### ISSUE TITLE
Reload not working

### ISSUE THREAD

**Issue by axsk** [JuliaWeb/Mux.jl#76]
When overwriting a already served app with the @app macro, as described in the readme, the server does not update :(
Anyone else got this problem?

Using Julia 1.0.1 on MacOS

**Comment by axsk** [comment]
``` julia
using Mux

@app test = (
  Mux.defaults,
  page(respond("<h1>Hello World!</h1>")))

serve(test)

@app test = (
  Mux.defaults,
  page(respond("<h1>Hello Reload!</h1>")))
```

is still showing Hello World

**Comment by axsk** [comment]
Is there something wrong on my end or can someone confirm this? Restarting Julia all over again really kills my productivity right now..

**Comment by axsk** [comment]
solved by #71  :)
